test/1_stdlib/VarArgs.swift

```diff
@@ -12,8 +12,8 @@ typealias CGFloat = Double
 #endif
 
 func my_printf(format: String, _ arguments: CVarArgType...) {
-  withVaList(arguments) {
-    vprintf(format, $0)
+  withVaList(arguments) { (vaList: CVaListPointer) in
+    vprintf(format, vaList)
```

Comment:
Technically, we don't need to include this change to get the test passing. However, it did made it easier to spot that `__gnuc_va_list` wasn't being mapped properly.

On the one hand, leaving it in will make it easier for other ports--if the `va_list` mappings don't apply they'll immediately see what needs to be added to `MappedTypes.def`. On the other, it isn't strictly necessary.